scvi/external/stereoscope/_model.py

```diff
@@ -0,0 +1,120 @@
+from scvi.model.base import BaseModelClass
+from scvi._compat import Literal
+
+from scvi.external.stereoscope._module import RNADeconv, SpatialDeconv
+from anndata import AnnData
+from scvi.dataloaders import ScviDataLoader
+from scvi.lightning import VAETask
+import numpy as np
+
+
+class RNAStereoscope(BaseModelClass):
+    """
+    Reimplementation of Stereoscope for deconvolution of spatial transcriptomics from single-cell transcriptomics.
+    
+    https://github.com/almaan/stereoscope.
+    Parameters
+    ----------
+    sc_adata
+        single-cell AnnData object that has been registered via :func:`~scvi.data.setup_anndata`.
+    use_cuda
+        Use the GPU or not.
+
```

Comment:
need docstring for model kwargs -- see the scvi.py code for how to link to the proper model class.